src/wmnds/patterns/cookies/cookies.js

```diff
@@ -0,0 +1,117 @@
+const cookies = () => {
+  const hideCookieBanner = () => {
+    const cookiesBanner = document.querySelector('.wmnds-cookies-banner');
+    cookiesBanner.style = 'display:none';
+  };
+
+  const setCookie = (cname, cvalue, exdays) => {
+    const d = new Date();
+    d.setTime(d.getTime() + exdays * 24 * 60 * 60 * 1000);
+    const expires = 'expires=' + d.toUTCString(); // eslint-disable-line prefer-template
+    const domain = 'domain=' + window.location.hostname; // eslint-disable-line prefer-template
```

Comment:
Good solution on grabbing the domain from location.hostname!
In theory this SHOULD map it to *.wmnetwork.co.uk and allow the use of the same cookie across all our services. 
Will have to test when it goes live.

Reply:
Yeah, in theory it makes sense but I couldn't test it further... Once it is merged I guess we can see if it works properly. 👍